pages/api/contact.js

```diff
@@ -0,0 +1,31 @@
+export default function (req, res) {
+  const nodemailer = require('nodemailer')
+  const transporter = nodemailer.createTransport({
+    // This is for mailtrap tester. Need to update to real SMTP settings, such as Gmail.
+    host: 'smtp.mailtrap.io',
```

Comment:
can we set this and the port to `process.env.SMTP_HOST` and `process.env.SMTP_PORT` respectively? That way we should be able to merge the PR and put our own settings into the env without requiring further code changes.

Reply:
Done